process/mock/preprocessorMock.go

```diff
@@ -97,11 +97,11 @@ func (ppm *PreProcessorMock) RequestTransactionsForMiniBlock(miniBlock *block.Mi
 }
 
 // ProcessMiniBlock -
-func (ppm *PreProcessorMock) ProcessMiniBlock(miniBlock *block.MiniBlock, haveTime func() bool) error {
+func (ppm *PreProcessorMock) ProcessMiniBlock(miniBlock *block.MiniBlock, haveTime func() bool) ([][]byte, error) {
 	if ppm.ProcessMiniBlockCalled == nil {
-		return nil
+		return nil, nil
 	}
-	return ppm.ProcessMiniBlockCalled(miniBlock, haveTime)
+	return nil, ppm.ProcessMiniBlockCalled(miniBlock, haveTime)
```

Comment:
You should modify the signature of the method ProcessMiniBlockCalled and then you simply return ppm.ProcessMiniBlockCalled(miniBlock, haveTime)